pymc3/tests/test_sampling.py

```diff
@@ -203,13 +221,24 @@ def test_return_inferencedata(self):
             assert result.posterior.sizes["chain"] == 2
             assert len(result._groups_warmup) > 0
 
-            # inferencedata without tuning
-            result = pm.sample(**kwargs, return_inferencedata=True, discard_tuned_samples=True)
+            # inferencedata without tuning, with prior
+            prior = pm.sample_prior_predictive()
+            result = pm.sample(
+                **kwargs,
+                return_inferencedata=True,
+                discard_tuned_samples=True,
+                idata_kwargs={"prior": prior}
+            )
```

Comment:
just adding in `idata_kwargs` to this test, as that wasn't previously covered by any test